content/resources/use-cases/sdm/index.md

```diff
@@ -0,0 +1,217 @@
+---
+title: The Australian Acoustic Observatory, Hoot Detective, & the impact of good data on Species Distribution Models 
+weight: 1
+---
+
+By Dr Andrew Schwenke & Dr Rob Clemens
+
+## Introduction
+
+The Australian Acoustic Observatory (A2O) is a continental-scale acoustic sensor
+network that provides data for hundreds of acoustic sensors across seven
+Australian ecoregions. This data is freely available to researchers, citizen
+scientists, and the general public, for use under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/) licence. More information
+about the A2O can be found in a paper published in [Methods in Ecology and Evolution (Roe at al., 2021)](https://acousticobservatory.org/wp-content/uploads/2021/07/A2O_Methods-in-Ecology-and-Evolution_2021.pdf).
```

Comment:
use the doi link - a wordpress link is unstable compared to a doi

(or if you're linking because the MEE paper is not open access, then link to the DOI version **as well**)